editor/editor_properties.cpp

```diff
@@ -125,7 +157,24 @@ void EditorPropertyText::set_string_name(bool p_enabled) {
 }
 
 void EditorPropertyText::set_secret(bool p_enabled) {
+	is_secret = p_enabled;
+	if (p_enabled) {
+		show_secret = memnew(Button);
+		show_secret->set_clip_text(true);
```

Comment:
The button has no text, so why?

Reply:
I was looking at other implementations with buttons, like the file selector https://github.com/IceflowRE/godot/blob/7af4db58194fcc0dabd2652d4262aa5ff952000b/editor/editor_properties.cpp#L666

Reply:
Yeah, that line is wrong too.